db/models.py

```diff
@@ -1 +1,22 @@
 from django.db import models
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description= models.TextField(blank=True)
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Rase, on_delete=models.CASCADE)
```

Comment:
There is a typo in the ForeignKey reference. It should be `Race` instead of `Rase`.